src/components/TodoComponents/Todt.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPenToSquare } from '@fortawesome/free-regular-svg-icons';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import styles from './Todo.module.css';
+
+
+interface TodoProps {
+    task: {
+        id: string;
+        task: string;
+        completed: boolean;
+    };
+    toggleComplete: (taskId: string) => void;
+    deleteTodo: (taskId: string) => void;
+    editTodo: (taskId: string) => void;
+}
+
+export const Todo: React.FC<TodoProps> = ({ task, toggleComplete, deleteTodo, editTodo }) => {
```

Comment:
insted of specifing the type to the todo component, its better to just give the props the type like this:

export const Todo = ({ task, toggleComplete, deleteTodo, editTodo }: TodoProps) => {